app/scripts/dimApp.i18n.js

```diff
@@ -16,7 +16,15 @@
           Equip: "Equip",
           Vault: "Vault",
           Vanguard: "Vanguard",
-          Reputation: "Reputation"
+          Reputation: "Reputation",
+          Super_cooldown: "Super cooldown",
+          Melee_cooldown: "Melee cooldown",
+          Grenade_cooldown: "Grenade cooldown",
+          tier_progress: "{{progress}} for {{tier}}",
+          About: "About",
+          'Support DIM': "Support DIM",
```

Comment:
Doesn't really matter. I have been keeping to the English word where possible, because `<span translate="Reputation"></span>` is pretty clear about what it's supposed to be like. But some others are longer or have replacements so it's not worth it, or like the cooldowns the key is built from a substitution.